views/js/controller/Delivery/monitoring.js

```diff
@@ -560,6 +528,50 @@ define([
                 }
             });
 
+            // column: pause
+            model.push({
+                id: 'pause',
+                label: __('Pause'),
+                transform: function(value, row) {
+                    var status;
+                    var inactive = true;
+
+                    if (row.state && row.state.status) {
+                        status = _status.getStatusByCode(row.state.status);
+                        inactive = !status || status.can.pause !== true;
+                    }
+
+                    return actionBtnTpl({
+                        icon: 'pause',
+                        type: 'pause',
+                        title: __('Pause session'),
+                        disabled: inactive ? 'disabled' : ''
+                    });
+                }
+            });
```

Comment:
we add a type to each column, if it is of an action type, it support an action callback or an array of callback. this way we generalize the action column concept so it can be anywhere instead of having to be always at the end.
 (The current option action should be preserved however for retrocompability.)